src/utils/logger_conf.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import traceback
+import time
+import logging as log
+
+DEFAULT_FORMATTER = log.Formatter('[ %(levelname)s ] %(message)s')
```

Comment:
may be add a timestamp?

Reply:
timestamp will be added in jenkins, do you need duplicate it?

Reply:
okay then